share/ruby-build/jruby-9.1.9.0

```diff
@@ -1,2 +1,2 @@
-require_java7
+require_java8
```

Comment:
Can we leave 9.0 and 9.1 definitions intact, and just update jruby 9.2+ definitions?

Reply:
Updated. Please confirm! :)